frontend/rust-lib/flowy-sqlite/Cargo.toml

```diff
@@ -20,12 +20,10 @@ r2d2 = "0.8.10"
 libsqlite3-sys = { version = ">=0.8.0, <0.24.0", features = ["bundled"] }
 scheduled-thread-pool = "0.2.6"
 error-chain = "=0.12.0"
-openssl = { version = "0.10.45", optional = true, features = ["vendored"] }
```

Comment:
AFAIK, We need the `vendored` feature to package the SSL library into AppFlowy in case some machines haven't installed it. cc @richardshiue

Reply:
Yeah agreed. While I think static linking will work, my concern is that the ubuntu repository's version isn't the newest and may potentially conflict. vendored pulls and compiles the newest version for all 3 platforms. Same thing goes for rocksdb which made the tests fail

Reply:
Just wanted to note this is not review-ready yet. Sorry, I forgot to mention it in the comments. I didn't make it as a draft because I wanted to see how it affects github actions.
It is an attempt to reduce build time for developers and the overall CI pipeline time in github actions.

Reply:
I know there are some problems with static linking and if it works it will probably only be for development building not release.

Reply:
Static linking reduces "appflowy-core-dev" task in my machine from 20 to 10 minutes in my machine from a clean state.

Reply:
> Static linking reduces "appflowy-core-dev" task in my machine from 20 to 10 minutes in my machine from a clean state.

10 - 20mis?  how long will it take to build the "appflowy-core-dev" task on your machine from scratch?

Reply:
From scratch about 25 minutes. The whole "appflowy-dev" task again from a clean state, takes about 43 minutes.

Reply:
And by clean I mean no cache and etc. I've tested it in a docker container. Also my machine is old.